pkgs/development/tools/xcbuild/default.nix

```diff
@@ -18,16 +18,14 @@ let
 in stdenv.mkDerivation {
   pname = "xcbuild";
 
-  # Once a version is released that includes
-  # https://github.com/facebook/xcbuild/commit/183c087a6484ceaae860c6f7300caf50aea0d710,
-  # we can stop doing this -pre thing.
-  version = "0.1.2-pre";
+  # The project has been archived on github, we build the last available revision
+  version = "archived";
```

Comment:
```suggestion
  version = "unstable-2019-11-20";
```

Reply:
Is that how it's usually done when a git(hub)-repo is archived?

Reply:
I don't see the need to emphasize the archival status.

Reply:
> Is that how it's usually done when a git(hub)-repo is archived?

Nothing special.